scripts/ci/avail-ext-doc/update_extension_list.py

```diff
@@ -20,13 +20,17 @@
 from jinja2 import Template  # pylint: disable=import-error
 import requests
 
-SCRIPTS_LOCATION = os.path.abspath(os.path.join('.', 'scripts'))
-
-AZURE_DOCS_CLI_REPO_PATH = os.path.join('.', 'azure-docs-cli')
+# After migration to OneBranch, clone azure-cli-extensions repo and azure-docs-cli repo are required.
+# Also standardizes the directory structure:
+# - $(System.DefaultWorkingDirectory)
+#   - azure-cli-extensions
+#   - azure-docs-cli
+AZURE_CLI_EXTENSIONS_REPO_PATH = os.path.abspath(os.path.join('.', 'azure-cli-extensions'))
```

Comment:
So onebranch changes cur repo from `.` to `azure-cli-extensions`? Is there any doc for this?

Reply:
On onebranch, I manually downloaded the azure-cli-extensions and azure-docs-cli using the git clone command, but ado will automatically download the azure-cli-extensions repo.
This is the root cause why the directory structure changes.
I think this is expected. If we manually download all repos through the git clone command on ADO, the structure will also change.